parties.go

```diff
@@ -5,40 +5,45 @@ import (
 	"github.com/invopop/gobl/org"
 )
 
+// Address defines the XML structure for KSeF addresses
 type Address struct {
 	CountryCode string `xml:"KodKraju"`
 	AddressL1   string `xml:"AdresL1"`
 	AddressL2   string `xml:"AdresL2"`
```

Comment:
Usually fields like "Address Line 2" are optional. Is and `,omitempty` required here also in the xml tag?

Reply:
It is not required and the same thing is with a few other fields. I have changes prepared for this and it will be changed in the next PR.

Reply:
Great! I thought that might be the case.